openmeter/entitlement/entitlement.go

```diff
@@ -107,6 +115,37 @@ type Entitlement struct {
 	Config []byte `json:"config,omitempty"`
 }
 
+// ActiveFromTime returns the time the entitlement is active from. Its either the ActiveFrom field or the CreatedAt field
+func (e Entitlement) ActiveFromTime() time.Time {
+	return defaultx.WithDefault(e.ActiveFrom, e.CreatedAt)
```

Comment:
nit: let's use lo

Reply:
See: https://github.com/samber/lo?tab=readme-ov-file#fromptror

Also I am thinking about making `defaultx.WithDefault` as an alias for `lo.FromPtrOr`.